azure-pipelines.yml

```diff
@@ -0,0 +1,24 @@
+steps:
```

Comment:
Can you include what queue it should run it?
I want as little as possible in Azure Pipelines

Reply:
See the definitions in PowerShell/PowerShell for reference.

Reply:
The idea would be to have only 1 YAML file that works for different build definition, where only the queue (Windows/Linux/Mac) is different and set in UI of the the build definition.

Reply:
You can use a matrix and run it 3 time in different queues if that is the intent.